docs/cmakelists.md

```diff
@@ -142,13 +143,17 @@ When defining your module, if you only support the Stable ABI after some point,
 you should use (for example for 3.11):
 
 ```cmake
-if(NOT "${SKBUILD_SABI_COMPONENT}" STREQUAL "")
-  python_add_library(some_ext MODULE WITH_SOABI USE_SABI 3.11 ...)
+if(NOT "${SKBUILD_SABI_VERSION}" STREQUAL "")
+  python_add_library(some_ext MODULE WITH_SOABI USE_SABI ${SKBUILD_SABI_VERSION} ...)
 else()
   python_add_library(some_ext MODULE WITH_SOABI ...)
 endif()
 ```
 
+If you have a lot of libraries, you can conditionally save these two items into
+a variable with `set(USE_SABI ${SKBUILD_SABI_VERSION})` and use it in all your
+`python_add_library` calls.
```

Comment:
````suggestion
`python_add_library` calls:

```
if(NOT "${SKBUILD_SABI_VERSION}" STREQUAL "")
  set(USE_SABI "USE_SABI ${SKBUILD_SABI_VERSION}")
endif()

python_add_library(some_ext MODULE WITH_SOABI ${USE_SABI} ...)
```
````